test/metabase/models/revision_test.clj

```diff
@@ -13,6 +15,17 @@
 
 (models/defmodel ^:private FakedCard :report_card)
 
+(use-fixtures :each (fn [thunk]
+                      (with-redefs [metabase.models.revision.diff/model-str->i18n-str (fn [model-str]
+                                                                                        (case model-str
+                                                                                          "Dashboard" (deferred-tru "Dashboard")
+                                                                                          "Card"      (deferred-tru "Card")
+                                                                                          "Segment"   (deferred-tru "Segment")
+                                                                                          "Metric"    (deferred-tru "Metric")
+                                                                                          "NonExistModel" (deferred-tru "NonExistModel")
+                                                                                          "FakeCard"      (deferred-tru "FakeCard")))]
+                        (thunk))))
```

Comment:
Does this mean "NonExistModel" and "FakeCard" will be included in the strings that need translation? Or are they excluded because this is in a test namespace?

Reply:
ah we just need it for the model dispatch, it can be included if the object is used .

Reply:
I don't understand. Is that a yes, or no?

For clarity I'm asking if it should be this instead, to avoid translators trying to translate "NonExistModel":
```clojure
(use-fixtures :each (fn [thunk]
                      (with-redefs [metabase.models.revision.diff/model-str->i18n-str (fn [model-str]
                                                                                        (case model-str
                                                                                          "Dashboard" (deferred-tru "Dashboard")
                                                                                          "Card"      (deferred-tru "Card")
                                                                                          "Segment"   (deferred-tru "Segment")
                                                                                          "Metric"    (deferred-tru "Metric")
                                                                                          "NonExistModel" "NonExistModel"
                                                                                          "FakeCard"      "FakeCard"))]
                        (thunk))))
```

Reply:
currently not, and yes we should not translate it. I'll update the code